change/@ni-spright-components-6dcca5fc-0353-4ba2-b325-63565d42b95e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Summary of the changes I pushed:
- Keeping the storybook file structure and naming aligned with the components. So story naming aligns to `chat-conversation`
- Made the storybook templates more consistent (use double quotes on attributes, use Enum references instead of hard coded strings)
- Followed the pattern of mapping column and made the chat-message stories internal. Users are expected to use chat-conversation with chat-message and chat-message is not standalone, so just document it inside of chat-conversation
- Removed most of the chat-message standalone matrix stories. We really only care about chat-conversation's containing chat-messages so focusing on those matrix stories.